core/src/trezor/lvglui/i18n/locales/de.py

```
@@ -1025,8 +1024,29 @@
+    "Startbildschirm",
     "Ändern",
-    "Hintergrundbild hinzugefügt",
-    "Möchtest du es jetzt als Hintergrundbild festlegen?",
+    "Als Sperrbildschirm",
+    "Startbildschirm",
+    "Vorschau",
+    "Hintergrundbild aus der OneKey App hinzufügen",
+    "Laden Sie ein Bild in Mein OneKey hoch > Wählen Sie Ihr OneKey-Gerät aus > Hintergrundbild hinzufügen.",
+    "Sammlung",
+    "Krypto-Netzwerk nicht gefunden? Versuche, dich über USB oder Bluetooth zu verbinden",
+    "Legen Sie eine 4–50-stellige PIN fest, um Ihr Gerät vor unbefugtem Zugriff zu schützen.",
```

Ruff (0.14.3)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)